CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Table of Contents
+
+- [Unreleased](#unreleased)
+- [1.0.0 - YYYY-MM-DD](#100---yyyy-mm-dd)
```

Comment:
@sirdeggen 
Not sure what is the idea of this field, but shouldn't there be a date and the actual version of ARC?
I think the ARC version would be the tag, not?

Reply:
@boecklim  can we resolve or fix this somehow? Then we can merge

Reply:
The idea is to note releases here. We just added unreleased details here. You would add a date when there is a new release.